modules/claims_api/app/workers/claims_api/flash_updater.rb

```diff
@@ -8,13 +8,19 @@ class FlashUpdater
     include Sidekiq::Worker
     include SentryLogging
 
-    def perform(user, flashes)
+    def perform(user, flashes, auto_claim_id: nil)
```

Comment:
This won't allow our Sidekiq job to be model agnostic so that the va.gov team can use it but the more I think about that the more I think they should just use the gem directly themselves.

Reply:
i couldn't think of a better way to do this but im game for any ideas. i made the auto_claim_id optional trying to allow anyone to use the job with just a little less functionality if an error occurs. Letting them use the gem directly sounds good though too :+1: